scripts/apps/index.js

```diff
@@ -50,5 +50,6 @@ export default angular.module('superdesk.apps', [
     'superdesk.apps.publish',
     'superdesk.apps.templates',
     'superdesk.apps.monitoring',
-    'superdesk.apps.profiling'
+    'superdesk.apps.profiling',
+    'superdesk.apps.editor3',
```

Comment:
This is part of core, so `superdesk.core.editor3`, and load it in `core` please, not apps.